index.html

```diff
@@ -262,8 +262,9 @@
           publisher: "ITU",
           date: "2013-07",
           href: "https://www.itu.int/rec/R-REC-BT.2035-0-201307-I"
-      },
-    };
+      }
+    }
+  }
```

Comment:
Is this supposed to go from 2 closing braces } to 3?

Anyway, we only need to lose the comma, I think. The closing semicolon is still fine.

(Although, JavaScript tries to guess where to insert missing semicolons so maybe it will just work anyway? Probably better to be explicit about the semicolon going there.)

Reply:
ReSpec would not work for me without the extra closing brace?  and VS Code gave me a syntax warning on the semicolon.

Reply:
*shrugs* Okay, fair enough.
I'll check again once it has landed.
LGTM